src/components/StatTrack/index.js

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import cx from 'classnames';
+
+import styles from './styles.styl';
+
+export default function StatTrack(props) {
+  const { className, statTrack, description } = props;
+
+  let value = statTrack.$instanceData.flavorObjective.progress/statTrack.$objective.completionValue;
+  value = +value.toFixed(2);
+
+  let desc = statTrack.$objective.progressDescription;
+
+  return (
+    <div key={desc} className={(description) ? styles.description : styles.extraInfo}>
```

Comment:
key={...} is only needed when you have a list of elements in a loop. https://reactjs.org/docs/lists-and-keys.html

It doesn't mean anything here and can be removed.

Reply:
Also, rather than passing through a prop like `description` to toggle the style, it's nicer to use the className to let the parent style it. See comment below for more :)